Tasks/GithubRelease/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "GithubRelease",
+  "main": "main.js",
+  "dependencies": {
```

Comment:
all the dependencies should have scan results available

Reply:
Added a task for this